packages/cli/src/commands/runtimes.ts

```diff
@@ -0,0 +1,18 @@
+import * as api from '../rest/api'
+import { BaseCommand } from './baseCommand'
+
+export default class Runtimes extends BaseCommand {
+  static hidden = false
+  static description = 'List all supported runtimes and dependencies.'
+  async run (): Promise<void> {
+    const { data: runtimes } = await api.runtimes.getAll()
```

Comment:
This will also have the node version under the dependencies section. If you want it to be used as a package json directly out of the box, you should filter out the `node` entry in the json